Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -695,18 +700,24 @@ public static void CreateUpdateRunSetFromJSON(RunsetExecutor runsetExecutor, Run
                     {
                         foreach (BusinessFlowExecConfig businessFlowConfig in runnerConfig.BusinessFlows)
                         {
+                            BusinessFlow bf = (BusinessFlow)FindItemByIDAndName<BusinessFlow>(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary cast to 'BusinessFlow'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4969846)